README.md

```diff
@@ -101,6 +101,19 @@ All of the roles internally have the notion of the following "steps" that are co
 
 Running a playbook will run the respective roles for each of these. If instead you want to run a subset you can use the operation argument as such `ansible-playbook -e operation=run`.
 
+#### JFR Hyperfoil
+
+To record Hyperfoil into JFR files we need to enable the options. It is possible to record on the controller and the agents separately, which with a different approach.
+To collect on the controller, when running the controller benchmark to start, pass the `-e hyperfoil_jfr=true` argument.
```

Comment:
I could only get this working with the change on https://github.com/Hyperfoil/hyperfoil_setup/pull/6.